styles/base/_reset.scss

```diff
@@ -0,0 +1,127 @@
+html,
```

Comment:
What I've been doing with my SCSS lately is splitting it into 3 sub folders:
- Base: all things that fall under set-up, such as setting the font-size on the HTML element or doing a CSS reset
- Components: all the smaller, individual pieces of our site such as buttons, headings and icons
- Layout: all things that fall under structure of the site such as a hero, footer or a section